dle/.env.example

```diff
@@ -79,7 +79,8 @@ ELASTICSEARCH_USER=elastic
 # In the entrypoint, whether to load Django data to Elasticsearch; whether to create an ES NLP pipeline with the BERT model (script doesn't exist yet);
 # and/or whether to load vectors from an external source (don't exist yet);
 # LOAD_DJANGO_TO_ES=True
-# USE_BERT=False
 # LOAD_VECTORS=False
 # If ES_AUTO_SYNC is True, elasticsearch-django will update the index as data is changed (including on initial ingest)
-ES_AUTO_SYNC=True
\ No newline at end of file
+ES_AUTO_SYNC=False
+# If USE_BERT is True, the BERT model will be downloaded if it doesn't exist
+USE_BERT=True
```

Comment:
Do we need to update our local `.env` files w/ this?

Reply:
`USE_BERT`, yes. That will try to install the model to your local machine. I haven't tried setting this up from scratch again to see exactly what order everything has to happen in. I think we probably want to stop using `ES_AUTO_SYNC` and manage all the Elastic index updates manually since we're only going to modifying data in bulk (e.g. run a new scrape, then ingest to Elastic). The auto-sync makes more sense when there are more erratic changes in the Django data. But could be convinced otherwise.